Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubectl/cmd/get.go

```diff
@@ -310,7 +328,21 @@ func RunGet(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string
 			}
 			lastMapping = mapping
 		}
-		if _, found := printer.(*kubectl.HumanReadablePrinter); found {
+		if resourcePrinter, found := printer.(*kubectl.HumanReadablePrinter); found {
+			resourceName := resourcePrinter.GetResourceName()
+			if resourceName == "" {
+				resourceName = mapping.Resource
```

Comment:
If I followed the logic above correctly, then `mapping` may be `nil` here and this line may panic.
I hope `f.PrinterForMapping(cmd, mapping, allNamespaces)` handles a `nil` value for `mapping`...

Before accessing some value from `mapping` we need to make sure it is not `nil`.